cmd/duffle/relocate.go

```diff
@@ -98,92 +106,78 @@ duffle relocate path/to/bundle.json --relocation-mapping path/to/relmap.json --r
 }
 
 func (r *relocateCmd) run() error {
-	bun, err := r.setup()
+	relMap := make(map[string]string)
+
+	rel, _, cleanup, err := r.setup()
```

Comment:
It looks like it's the only call to `r.setup()`, and the second argument is dropped, so I guess it can safely been removed from the returned arguments?

Reply:
Yes, well spotted. That was a vestige of the earlier form of relocation which produced a modified bundle. Fixed.